### ISSUE TITLE
Print the bottom geometry designs

### ISSUE THREAD

**Issue by Kanha-Matai** [AguaClara/HRS-Bot-Geo#4]
After determining the designs we believe should work for the bottom geometries, we will attempt to 3D print them for testing.

**Comment by monroews** [comment]
Could you share an update on your progress on the bottom geometry? I'd like to see your ideas with the goal of moving this along to printing and testing as soon as possible.

**Comment by tm545** [comment]
Hi, I'm from the fluoride auto sub team and we had some of the same issues as you guys might have had so I just threw together an CAD file or a design based on a meeting we had with Monroe today. If you would like to ask me anything or if this design wouldn't work please reach out to us.

The design was based on the OD of the reactor followed by linear wedges that are 60 degrees in the XY plane and then a freefall section followed by another wedge in the ZY plane also in the 60 degrees to reach the OD of the tubing below the reactor.
https://a360.co/2DTLGGt (that link will show you my autodesk file you can open with OnShape)

**Comment by monroews** [comment]
The wedge in the ZY plane is not constant width and thus will clog. That plane needs to be flat and constant width. The width of that plane should be 0.17 inch to match the diameter of the incoming jet.

**Comment by Kanha-Matai** [comment]
Hello,
We have been working at trying to 3D draw these designs. Currently, we are looking to create conical shapes which vary in slopes along the sides. We believe that this would mitigate the "group hug" effect that is present as the varying velocities would allow for particles to fall in and not get clogged.

I have uploaded an image to our repository showing the current position. I have been struggling to figure out how to connect the curve to the center point, something I have been trying to solve for the past week.

In regards to the printing itself, we have obtained access to the Rapid Prototyping Lab which has the best 3D-printer available to students and project teams. We will be going through a training session the day we are back from Fall break.

**Comment by Kanha-Matai** [comment]
Would you recommend redrawing the design provided by the Flouride team with the fixed zy plane?

**Comment by monroews** [comment]
Yes, that would be excellent. Note that the fluoride team's drawing is close, but was not yet using planes for  all of the sloped surfaces.

**Comment by tm545** [comment]
Hi again,
I played around with our Autodesk Drawing and finished with design https://a360.co/2IF2IqJ . This one has the Monroe handshake of approval but still needs to be printed and experimented with to make sure all of our ideas were correct (linear wedges at 60 degrees won't allow flocs to aggregate). There are some parameters you can change like the freefall length (distance between the two wedges) and the width of the material on the inside (the stuff that isn't clear) and the width of the material on the outside (that cylinder that is clear) based off of whatever RPL tells you it can print with. I can work on adding more parameters if you need but just let me know what you're looking for.
Best,
Fluoride Auto

**Comment by Kanha-Matai** [comment]
Hello Tigram and all,
Thank you for your help and efforts with the drawing. We will get this design printed as soon as possible and will begin testing soon.

**Comment by tm545** [comment]
Could you print us a copy as well?

**Comment by tm545** [comment]
Hi again thanks for the new Bot Geo. We got ours into the reactor but its not as watertight as we had imagined so we put some teflon tape on the bottom of the printed device to make sure it, it didn't keep moving up the reactor. We might also need to fill in the bottom just to make sure the influent tube doesn't get jammed in the bottom of the print. We can either just tape that or if you're feeling like printing a new one I can CAD that just let me know what you'd like to do.

**Comment by tm545** [comment]
https://a360.co/2IF2IqJ

**Comment by tm545** [comment]
That;s a CAD with the bottom
